src/utils.rs

```diff
@@ -118,6 +213,109 @@ impl<Manager> Utils<Manager> {
             sys::SteamAPI_ISteamUtils_SetWarningMessageHook(self.utils, Some(c_warning_callback));
         }
     }
+
+    /// Gets the gamepad text input from the Big Picture overlay.
+    ///
+    /// This must be called within the `GamepadTextInputDismissed_t` callback, and only if
+    /// `GamepadTextInputDismissed_t::m_bSubmitted` is true.
+    ///
+    /// Provides the text input as UTF-8.
+    pub fn get_entered_gamepad_text_input(&self) -> Option<String> {
+        unsafe {
+            let mut buf = [0u8; 1024];
+            let len = 0;
+            let res = sys::SteamAPI_ISteamUtils_GetEnteredGamepadTextInput(
+                self.utils,
+                buf.as_mut_ptr() as *mut i8,
+                buf.len() as u32,
+            );
+            if res {
+                Some(String::from_utf8_lossy(&buf[..len as usize]).into_owned())
```

Comment:
This doesnt look right to me, `len` is never assigned

Reply:
You're right. I'm gonna add a parameter for that.